index.html

```diff
@@ -4,7 +4,67 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>Kevin Weyrowitz</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <ul>
+        <li><a href="#Contact">Contact</a></li>
+        <li><a href="#Biography">Biography</a></li>
+      </ul>
+    </header>
+    <main>
+      <h1>Kevin Weyrowitz</h1>
+      <article>
+        <img src="https://bit.ly/3ljrST5" alt="alternatives bild" />
+        <p>This is me in the future doing interesting thinks :)</p>
+      </article>
+      <hr />
+      <article id="Contact">
+        <p>Num: 0000/0000000</p>
+        <a href="kevinweyrowitz93@gmail.com">kevinweyrowitz93@gmail.com</a>
```

Comment:
switch href to href="mailto:kevinweyrowitz93@gmail.com"

Reply:
thx :)